perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupStructureSynchronizationIntegrationTest.java

```diff
@@ -93,7 +93,6 @@ public void setUpBeforeEveryMethod() throws Exception {
 		MockitoAnnotations.initMocks(this);
 
 		doReturn(essa).when(extSourceManagerBl).getExtSourceByName(any(PerunSession.class), any(String.class));
-		doReturn(EXT_SOURCE_NAME).when((ExtSourceLdap)essa).getName();
```

Comment:
This should probably be `doReturn(EXT_SOURCE_NAME).when(((ExtSourceLdap)essa).getName());`